jac/jaclang/compiler/jac.lark

```diff
@@ -43,7 +43,7 @@ architype: decorators? architype_decl
          | architype_def
          | enum
 
-architype_decl: arch_type access_tag? STRING? NAME inherited_archs? (member_block | SEMI)
+architype_decl: KW_ASYNC? arch_type access_tag? STRING? NAME inherited_archs? (member_block | SEMI)
```

Comment:
Lets bring this up to the architype production rule, and check for only walkers and node and edges.